.github/workflows/cd.branch-test.yml

```
@@ -3,7 +3,7 @@ name: branch-test - Continuous Deployment
 
 on:
   push:
-    branches: [some-branch] # <-- adjust for current branch to test
+    branches: [exam/new-background-color]  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)